twitter_ads/creative.py

```diff
@@ -595,3 +596,37 @@ def all(klass, account, **kwargs):
         resource = klass.RESOURCE_COLLECTION.format(account_id=account.id)
         request = Request(account.client, 'get', resource, params=kwargs)
         return Cursor(None, request)
+
+
+class Card(Resource):
+
+    PROPERTIES = {}
+
+    RESOURCE_COLLECTION = '/' + API_VERSION + '/accounts/{account_id}/cards'
+
+    def create(klass, account, name, components):
+        method = 'post'
+        resource = klass.RESOURCE_COLLECTION.format(account_id=account.id)
+        headers = {'Content-Type': 'application/json'}
+        payload = {'name': name, 'components': components}
+        response = Request(
+            account.client, method, resource,
+            headers=headers, body=json.dumps(payload)
+        ).perform()
+        return response.body['data']
+
+    def load(klass):
+        raise AttributeError("'Card' object has no attribute 'load'")
+
+    def reload(klass):
+        raise AttributeError("'Card' object has no attribute 'reload'")
+
+# card properties
+# read-only
```

Comment:
Not including `id` or `card_type` as these were deprecated in [v9](https://twittercommunity.com/t/ads-api-version-9/150316). That being said, I can add this in if we plan to roll this change out with v8 of the SDK.

Reply:
@tushdante what are your thoughts here?